docs/python/getting-started.md

```diff
@@ -0,0 +1,180 @@
+# Getting Started with Python Test Acceleration
```

Comment:
It seems like this file got copied instead of being renamed? It's hard for me to tell if anything changed here.

If you're changing the URL for this, you should check with @amyhua because I think the marketing website has some links into this document that would need to be updated.

Reply:
Oops, yeah this was meant to be a move. I'll replace the old version with a meta redirect so no links break.